README.md

```diff
@@ -34,19 +34,21 @@ The week numbers correspond to the weeks of the school semester. We meet weekly
 - Week 3: Coding and Arduino (lab 1)
```

Comment:
regarding the directory name changes, I think it would look cleaner if we stayed consistent with our "-" and "_" usage , and the capitalization. I prefer underscores, and all lowercase, so I propose: 
- lab1_getting_started_with_arduinos
- lab2_first_arduino_sensor
- lab3_missions
- lab4_intro_to_toastsat_sensors
- lab5_sd_card
- lab6_toastsat_design_considerations
- lab7_fall_semester_wrap_up

This is just a personal preference though, so very willing to change this as long as we stay consistent.

Reply:
yes I did not realy like what I did.  you are correct.